src/common/PATCHES.js

```diff
@@ -0,0 +1,16 @@
+const PATCHES = [
+  {
+    name: '8.0',
+    timestamp: 1534197600000,
```

Comment:
Could you leave a comment somewhere with which region/tmz you are using for these timestamps? For maintainability on following patches.